v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-ibc.md

---
index: 15
title: What is IBC?

> description: IBC is an acronym for Inter-Blockchain Communication Protocol, which is a protocol for secure message passing between heterogeneous blockchains.

Reviewer: ```suggestion
description: IBC is an acronym for Inter-Blockchain Communication protocol, a protocol for passing secure messages between heterogeneous blockchains.
```

Reviewer: I think this is accurate, double check and let met know, I'm learning

Reviewer 2: It’s accurate

  - blockchain
  - atom
---

> IBC is an acronym for Inter-Blockchain Communication Protocol, which is a protocol for secure message passing between heterogeneous blockchains. It is a central piece of the Cosmos vision, as it will enable blockchains to interract in ways that were not possible before. For a deeper look at IBC, please refer to the [IBC documentation](https://ibcprotocol.org/documentation).

Reviewer: ```suggestion
IBC is an acronym for IBC is an acronym for Inter-Blockchain Communication protocol, a protocol for passing secure messages between heterogeneous blockchains. IBC is a central piece of the Cosmos vision and enables blockchains to interact in ways that were not possible before. For a deeper look at IBC, see the [IBC documentation](https://ibcprotocol.org/documentation).
```
